### PR TITLE
Added force unconfirm/confirm.

### LINKED ISSUE #24: Admin Dashboard - Force confirm/unconfirm user's email
The admin can force the confirmation of a user or unconfirm him.

### PR DESCRIPTION
**Pull request by Seluj78**
Signed-off-by: Jules Lasne (jlasne) <jlasne@student.42.fr>

Fixes #24

## Proposed Changes

  - Added two buttons that appear differently based on the user's confirmation status
  - Added the routes to confirm/unconfirm the user
  -

### CODE REVIEW

File: openapprentice/routes/admin/user.py [hunk 1]
```diff
@@ -115,3 +116,49 @@ def admin_promote_user(uuid):
...
+    # TODO: Checkbox to send email to notify user?
+
+    :param uuid: The uuid of the user to unconfirm
+    """
+    user = get_user(uuid)
+    if not user.is_confirmed:
+        flash(gettext("This user is already unconfirmed"), "error")
+    else:
+        if user.uuid == current_user.uuid or user.email == "admin@openapprentice.org":
+            flash(gettext("You can't confirm yourself"), "error")
```
Comment (Vigasaurus) [inline openapprentice/routes/admin/user.py]: You can't **unconfirm yourself.

Comment (Seluj78) [reply openapprentice/routes/admin/user.py]: Oh yeah nice typo find ! i'll correct it asap.

**Comment by Seluj78** [comment]
@Vigasaurus What's you wording idea ? This is the admim dashboard

**Comment by Vigasaurus** [comment]
I don't know yet, I'll let you know when I think of something.

**Comment by Seluj78** [comment]
Change is done @Vigasaurus.
Alright, let me know

**Comment by Vigasaurus** [comment]
Maybe if we change both, to: 'authorize' and 'revoke authorization'

**Comment by Seluj78** [comment]
Well it's supposed to be a confirmation based on your email, like is your account verified/confirmed

**Comment by Seluj78** [comment]
`Authorize` and `revoke` I would use for the API access :smile:

**Comment by Vigasaurus** [comment]
Oh haha then, confirm and deny is fine I think. Will let you know if I think of something better.

**Comment by Seluj78** [comment]
I don't like the `deny` hence it's not denying someone, it's un confirming his account

**Comment by Vigasaurus** [comment]
Yeah, I guess unconfirm is fine then. Because this is only admin facing right, no user would see that?

**Comment by Seluj78** [comment]
Yes this is admin only

**Comment by Seluj78** [comment]
@Vigasaurus can you confirm your review on the PR now that we agreed ?

**Pull request merged**